Ontology/Asset/Equipment/ICTEquipment/GatewayEquipment.json

```diff
@@ -0,0 +1,56 @@
+{
+  "@id": "dtmi:digitaltwins:rec_3_3:asset:GatewayEquipment;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Gateway"
+  },
+  "extends": "dtmi:digitaltwins:rec_3_3:asset:ICTEquipment;1",
+  "contents": [
```

Comment:
Similar to Controller, I would remove all of these Properties.

Reply:
Done